ops/friends-deploy.tmpl.yaml

```diff
@@ -60,9 +60,9 @@ ingress:
   }
   tls:
     - hosts:
-        - "staging.hykuup.com"
-        - "*.staging.hykuup.com"
-      secretName: staging-tls
+        - "hykuup-knapsack-friends.notch8.cloud"
+        - "*.hykuup-knapsack-friends.notch8.cloud"
```

Comment:
Is it not possible to keep the staging.hykuup.com host on the friends cluster?

Reply:
It is, we just need to figure out how to set it up dynamically.